server.py

```diff
@@ -174,14 +174,11 @@ def get_verified_email(token, nonce):
     #  - Check that expiration time is not in the past
     #    (The `exp` claim in the payload).
     #
-    #  - Check that the 'not before' time is not in the future
```

Comment:
This should also probably get squashed into previous iterations of this comment.